### PR TITLE
WIP: Rhino assessee guidance

### PR DESCRIPTION
**Pull request by adrianclay**

### CODE REVIEW

File: core-skills/frontend-web-development/rhino-assessee.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
+
+## Semantic HTML
+
+The first stage is an assessment of your ability to use semantic HTML.
```
Comment (kjdchapman) [inline core-skills/frontend-web-development/rhino-assessee.md]:
```suggestion
```

I don't think this sentence adds anything.

File: core-skills/frontend-web-development/rhino-assessee.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
```
Comment (adrianclay) [inline core-skills/frontend-web-development/rhino-assessee.md]: Give some indication as to how long you need to set aside for this assessment.

File: core-skills/frontend-web-development/rhino-assessee.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Semantic HTML
+
+
+We have [prepared a HTML page][rhino-semantic-html-page],
+and you will need to change the HTML for more semantic version.
+
+Replace the existing tags with 5 more semantic ones.
```
Comment (adrianclay) [inline core-skills/frontend-web-development/rhino-assessee.md]: Odd wording.

File: core-skills/frontend-web-development/rhino-assessee.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+For each change, explain what meaning you thought the original HTML was to convey, and why you chose the semantic tag.
+
+[rhino-semantic-html-page]: https://codepen.io/kjdchapman/pen/zXzQYw
+
+## Basic CSS
+
+You're provided with some [unstyled HTML][rhino-unstyled-html-page],
+which you'll need to add styles such that it looks like the below screenshot.
```
Comment (adrianclay) [inline core-skills/frontend-web-development/rhino-assessee.md]: Pixel perfect isn't important.

File: core-skills/frontend-web-development/rhino-assessee.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
+
+## Semantic HTML
+
+
+We have [prepared a HTML page][rhino-semantic-html-page],
```
Comment (adrianclay) [inline core-skills/frontend-web-development/rhino-assessee.md]: You can bring your own version if desired.

File: core-skills/frontend-web-development/rhino-assessee.md (hunk 2 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/rhino-assessee.md]: Addressed in 07558b6

File: core-skills/frontend-web-development/rhino-assessee.md (hunk 3 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/rhino-assessee.md]: Significantly reworded this in 07558b6.

File: core-skills/frontend-web-development/rhino-assessee.md (hunk 5 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/rhino-assessee.md]: Let's talk about exactly what this means - I've cleaned up the surrounding content a little (https://github.com/madetech/learn/pull/119/commits/f92d877f9fe14a423e57ad5c2b9a7d59c78ab9f0) but I'm still getting my head around what can/should be pre-prepared by an candidate.

File: core-skills/frontend-web-development/rhino-assessee.md (hunk 4 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/rhino-assessee.md]: Resolved in https://github.com/madetech/learn/pull/119/commits/f92d877f9fe14a423e57ad5c2b9a7d59c78ab9f0

**Pull request merged**